packages/squiggle-lang/__tests__/TS/Symbolic_test.ts

```diff
@@ -12,7 +12,7 @@ describe("Symbolic mean", () => {
             expect(squiggleResult.value).toBeCloseTo((x + y + z) / 3);
           } catch (err) {
             expect((err as Error).message).toEqual(
-              "Expected squiggle expression to evaluate but got error: TODO: Triangular values must be increasing order."
+              "Expected squiggle expression to evaluate but got error: Math Error: Triangular values must be increasing order."
```

Comment:
I think we can do better than `Math Error`. It's about validity of input, the _user_ messed something up. Whereas `Math Error` sounds like _the internals_ messed something up.

Reply:
Chatted with Ozzie and decided that we would like to improve error messages in another story